great_expectations/core/usage_statistics/usage_statistics.py

```diff
@@ -566,7 +571,55 @@ def get_checkpoint_run_usage_statistics(
             )
         except Exception as e:
             logger.debug(
-                f"{UsageStatsExceptionPrefix.EMIT_EXCEPTION.value}: {e} type: {type(e)}, get_batch_list_usage_statistics: Unable to create anonymized_checkpoint_run payload field"
+                f"{UsageStatsExceptionPrefix.EMIT_EXCEPTION.value}: {e} type: {type(e)}, get_checkpoint_run_usage_statistics: Unable to create anonymized_checkpoint_run payload field"
+            )
+
+    return payload
+
+
+def get_profiler_run_usage_statistics(
+    profiler: "RuleBasedProfiler",  # noqa: F821
+    variables: Optional[dict] = None,
+    rules: Optional[dict] = None,
+    *args,
+    **kwargs,
+) -> dict:
+    usage_statistics_handler: Optional[
+        UsageStatisticsHandler
+    ] = profiler._usage_statistics_handler
+
+    data_context_id: Optional[str] = None
+    if usage_statistics_handler:
+        data_context_id = usage_statistics_handler._data_context_id
+
+    anonymizer: Optional[Anonymizer] = _anonymizers.get(data_context_id, None)
+    if anonymizer is None:
+        anonymizer = Anonymizer(data_context_id)
+        _anonymizers[data_context_id] = anonymizer
+
+    payload: dict = {}
+
+    if usage_statistics_handler:
+        # noinspection PyBroadException
+        try:
+            profiler_run_anonymizer: "ProfilerRunAnonymizer" = (  # noqa: F821
+                usage_statistics_handler._profiler_run_anonymizer
+            )
+
+            resolved_runtime_config: "RuleBasedProfilerConfig" = (  # noqa: F821
+                RuleBasedProfilerConfig.resolve_config_using_acceptable_arguments(
+                    profiler=profiler,
+                    variables=variables,
+                    rules=rules,
+                )
+            )
+
+            payload: dict = profiler_run_anonymizer.anonymize_profiler_run(
```

Comment:
@cdkini positional vs. keyword arguments... I prefer keyword ones for easier readability (just a thought)...

Reply:
will change in the next commit!